setup.py

```diff
@@ -72,7 +72,7 @@
         'test' : ['pytest', 'pytest-cov', 'numpy', 'matplotlib', 'scipy', 'recommonmark'],
         'demo' : ['jupyter', 'jupyter_client', 'ipython'],
         'doc' : ['sphinx'],
-        '10x': ['pathos', 'pysam']
+        '10x': ['bam2fasta']
```

Comment:
We can specify the dependency here as `bam2fasta==1.0.1`, and update as needed in the future.